udata/harvest/tests/test_dcat_backend.py

```diff
@@ -240,6 +240,19 @@ def test_harvest_big_catalog(self, rmock):
         actions.purge_jobs()
         assert get_from_json(current_app.config.get('HARVEST_GRAPHS_S3_BUCKET'), job.data['filename']) is None
 
+    @pytest.mark.options(SCHEMA_CATALOG_URL='https://example.com/schemas', HARVEST_MAX_ITEMS=2)
```

Comment:
Thank you for this test :pray:
Could you add some similar test on ckan if it doesn't exist already?

Reply:
Done in https://github.com/opendatateam/udata-ckan/pull/251/commits/7063ada45d7d54592eda0bb61caf1c637c5a974b